terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,56 @@
 provision_custom_search = False
 
 
+def content_security_policy() -> str:
+    if config.deployment_stage == 'anvildev':
+        broad_deployment = 'dev'
+    elif config.deployment_stage == 'anvilprod':
+        broad_deployment = 'prod'
+    else:
+        assert False, config.deployment_stage
+    return ' '.join([
```

Comment:
This is still rather ugly because it doesn't follow through with the technique to its logical end. 

A CSP is a string that follows a well-defined [grammar](https://w3c.github.io/webappsec-csp/#grammardef-serialized-policy). That grammar uses two separators: semicolons and spaces. This means you should use `' '.join(…)` and `';'.join(…)` to mirror the productions in the grammar. We did this before when we were composing Cloudwatch Metric queries so you should already be familiar with this technique.